system/security/shared/caching.cpp

```diff
@@ -556,6 +556,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (!fullScope || !*fullScope)
     {
         *accessFlags = queryDefaultPermission(sec_user);
+        WARNLOG("FileScope unspecified for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
```

Comment:
Why not use PROGLOG? Is it possible to use the log level control here? (same as other new WARNLOGs below).

Reply:
I discussed with Russ before he left. We both felt that while it's not an error, it's something that warrants attention and should probably be fixed. 

The PROGLOG below is because it is desirable to always have that message appear for researching denials.